src/editpage/components/assistance/SettingsModal.vue

```diff
@@ -174,6 +191,17 @@ export default {
       this.updateConfigurationString();
     },
 
+    exportLogs: function() {
+      const {dialog} = require('electron').remote;
+      const options = {defaultPath: 'logs.zip'};
+      const targetDir = dialog.showSaveDialogSync(options);
+      const electron = require('electron');
+      const sourceDir = (electron.app || electron.remote.app).getPath(
+          'userData') + '\\activity-logs';
+      const zipper = require('zip-local');
```

Comment:
Since we use `zip-local` lets add it to the package.json
Also why did you pick `zip-local`?

Reply:
Out of the different packages that I tried for zipping files, it was the one that I managed to get working easily. If preferred, I could work on getting a different package to work.